src/c++/perf_analyzer/client_backend/client_backend.h

```diff
@@ -189,6 +156,12 @@ class ClientBackendFactory {
   /// \param backend Returns a new Client backend object.
   Error CreateClientBackend(std::unique_ptr<ClientBackend>* backend);
 
+  /// Add library path and model repository path to start the server and load
+  /// the model only used for the TRITON_LOCAL version which uses CAPI
+  Error AddAdditonalInfo(
```

Comment:
How is url_, etc. passed into the client backends? You should just follow that same pattern and pass model repository and server library path into triton_c_api backend and it can then use them as appropriate.